client/src/components/WorkflowInvocationState/WorkflowInvocationOverview.vue

```diff
@@ -28,6 +29,7 @@ function getUrl(path: string): string {
 
 interface Props {
     invocation?: WorkflowInvocationElementView;
+    workflow?: Workflow;
```

Comment:
I think it was cleaner, more performant and self-contained before ? `useWorkflowInstance` is great, it only fetches the workflow if we don't already have it, we now seem to always fetch the workflow even if it is already in store.